### PR TITLE
Update /esm page

### LINKED ISSUE #4794: [COPY UPDATE] esm - "Ubuntu Extended Security Maintenance" - UA for Infra
Please update esm - "Ubuntu Extended Security Maintenance" from the [copy doc](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update the /esm page, mostly the FAQ section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/esm](http://0.0.0.0:8001/esm)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit#)

## Issue / Card

Fixes #4794

### CODE REVIEW

File: templates/esm/index.html [hunk 1]
```diff
@@ -136,28 +134,25 @@ <h4 class="u-no-max-width">Which hardware platforms are supported under Ubuntu E
       <p>Currently, we are maintaining the Ubuntu Cloud/Server 64-bit AMD/Intel binaries. We will extend support for other platforms in future updates.</p>
 
       <h4 class="u-no-max-width">Do all levels of Ubuntu Advantage have access to Ubuntu ESM?</h4>
```
Comment (steverydz) [inline templates/esm/index.html]: According to the copy doc this should be:

```
```

File: templates/esm/index.html [hunk 2]
```diff
@@ -136,28 +134,25 @@ <h4 class="u-no-max-width">Which hardware platforms are supported under Ubuntu E
...
+      <h4 class="u-no-max-width">How can we ensure the security of our Ubuntu systems after the end of Basic Security Maintenance?</h4>
+      <p>Sign up to Ubuntu Advantage now, and you will benefit from UA services immediately without having a gap in service when a particular release reaches end-of-life. For more details of initial release dates, and hence release end-of-life dates, see <a href="https://wiki.ubuntu.com/Releases">wiki.ubuntu.com/Releases</a>). Ubuntu Advantage is available at <a class="p-link--external" href="https://buy.ubuntu.com/">buy.ubuntu.com</a>, and or for AWS users, you can purchase Ubuntu Advantage <a class="p-link--external" href="https://aws.amazon.com/marketplace/pp/B01MZ1LSBQ?qid=1486382973569&sr=0-1&ref_=srh_res_product_title">Standard</a> or <a class="p-link--external" href="https://aws.amazon.com/marketplace/pp/B01N9JG6EM?qid=1490357851557&sr=0-7&ref_=srh_res_product_title">Advanced</a> through the AWS Marketplace.</p>
```
Comment (steverydz) [inline templates/esm/index.html]: There is no opening parentheses for the wiki.ubuntu.com/releases link

File: templates/esm/index.html [hunk 3]
```diff
@@ -136,28 +134,25 @@ <h4 class="u-no-max-width">Which hardware platforms are supported under Ubuntu E
...
-
-      <h4 class="u-no-max-width">We're mirroring the repository on our internal Landscape Server. Is there a guide on how to get Ubuntu ESM if using Landscape?</h4>
+      <h4 class="u-no-max-width">We’re mirroring the repository on our internal Landscape Server. Can we still get Ubuntu ESM if using Landscape?</h4>
       <p>ESM is just a regular Ubuntu archive, but authenticated and served over HTTPS. Archive mirroring is already available in Landscape, and is the only supported mechanism for mirroring the ESM archive.</p>
 
       <h4 class="u-no-max-width">What will the LTS support situation look like during ESM? Will we be able to raise functionality tickets with Canonical or are you ending actual support and providing only security patches?</h4>
-      <p>The support window for Ubuntu 12.04 closes on April 28, 2017. The support team will not be able to fix bugs or build fixed packages once the 12.04 LTS archive is closed.</p>
+      <p>The support window for Ubuntu 12.04 LTS closes on April 28, 2017. The support team will not be able to fix bugs or build fixed packages once the 12.04 LTS archive is closed.</p>
```
Comment (steverydz) [inline templates/esm/index.html]: This "LTS" isn't in the copy doc, but I think it makes sense to include it here

Comment (pmahnke) [reply templates/esm/index.html]: I updated the copy doc to reflect this.

**Pull request merged**